[WPE][GTK] Missing Soup module documentation links

siwei-li opened the pull request:
#### 6dfebc18131711b36782ae8aa527d246b8919764
<pre>
[WPE][GTK] Missing Soup module documentation links
<a href="https://bugs.webkit.org/show_bug.cgi?id=242900">https://bugs.webkit.org/show_bug.cgi?id=242900</a>

Reviewed by Michael Catanzaro.

* Source/WebKit/gtk/urlmap.js.in: Point &quot;Soup&quot;(libsoup-3.0) to <a href="https://libsoup.org/libsoup-3.0/">https://libsoup.org/libsoup-3.0/</a>
* Source/WebKit/wpe/urlmap.js: Ditto

Canonical link: <a href="https://commits.webkit.org/252982@main">https://commits.webkit.org/252982@main</a>
</pre>

mcatanzaro reviewed (approved):
Nice, thanks. Please alphabetize it, though.

siwei-li committed:
[WPE][GTK] Missing Soup module documentation links
https://bugs.webkit.org/show_bug.cgi?id=242900

Reviewed by Michael Catanzaro.

* Source/WebKit/gtk/urlmap.js.in: Point "Soup"(libsoup-3.0) to https://libsoup.org/libsoup-3.0/
* Source/WebKit/wpe/urlmap.js: Ditto

Canonical link: https://commits.webkit.org/252982@main
